en-US/main.lang

```diff
@@ -309,14 +309,13 @@ Portions of this content are ©1998–%(current_year)s by individual mozilla.org
 Portions of this content are ©1998–%(current_year)s by individual mozilla.org contributors. Content available under a <a href="%(url)s">Creative Commons license</a>.
 
 
-# Footer string (will replace "Contribute to this page")
+# Footer string
 ;Contribute to this site
 Contribute to this site
 
 
-# Footer string
-;Contribute to this page
-Contribute to this page
+;Source code for this page
```

Comment:
Good call on removing the other string, but I would add a comment, e.g. "Used in footer, links to the source code of the page currently displayed".